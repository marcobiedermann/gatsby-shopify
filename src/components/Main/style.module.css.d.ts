declare const styles: {
  readonly main: string;
};
export = styles;
