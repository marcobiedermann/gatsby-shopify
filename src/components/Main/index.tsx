import React, { FC } from 'react';
import styles from './style.module.css';

const Main: FC = (props) => <main className={styles.main} {...props} />;

export default Main;
