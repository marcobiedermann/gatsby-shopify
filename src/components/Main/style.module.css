.main {
  padding-bottom: 6em;
  padding-top: 6em;
}
