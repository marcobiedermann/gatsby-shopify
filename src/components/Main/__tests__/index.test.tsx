import { render } from '@testing-library/react';
import React from 'react';
import Main from '..';

describe('main component', () => {
  it('should render correctly', () => {
    const { container } = render(<Main>Main</Main>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
