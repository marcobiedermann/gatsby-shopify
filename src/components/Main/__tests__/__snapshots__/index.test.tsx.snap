// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main component should render correctly 1`] = `
<main
  class="main"
>
  Main
</main>
`;
